platform/upgrades/adding-custom-cloud-credentials.md

```diff
@@ -318,32 +318,24 @@ grid delete cluster <cluster name>
 
 ## Cost saving mode
 
-There are two cluster management modes you can pick, depending on your expected cluster size and latency/cost preferences.
-They are easily switched using the `--cost-savings` flag when creating the cluster.
+You can turn on cost saving mode to lower the cluster cost, but it might affect start up latency. Depending on the region these costs are around ~$10/day, compared to ~$50/day for the default mode.
 
-* default(performance)
+You can use `--cost-saving` flag when starting a new cluster:
 
-```
-  "performance_profile": "CLUSTER_PERFORMANCE_PROFILE_DEFAULT",
+```bash
+grid clusters aws --role-arn $ROLE_ARN --external-id $EXTERNAL_ID --region us-west-2 --cost-savings --instance-types t2.medium,t2.large <cluster name>
 ```
 
-* cost saving:
+Or edit an existing cluster (note: you might need to switch smaller nodes)
 
+```bash
+grid edit cluster <cluster name> --cost-savings --instance-types t2.medium,t2.large
```

Comment:
hmmmm this does not work like this nowadays. Have you tried `grid edit cluster` command?  It's similar to `kubectl edit <resource> name`, it opens up the resource config in $EDITOR and let's you change fields there.